tests/cli/test_integration_datasets.py

```diff
@@ -1293,14 +1293,56 @@ def test_renku_clone_with_config(tmpdir):
             }
         )
 
-        repo = git.Repo('datasets-test')
+        assert 'master' == str(repo.active_branch)
```

Comment:
I'd rather use `repo.active_branch.name` instead of relying on `str()` function.

Reply:
Done.